src/models/reposyncstate.ts

```diff
@@ -208,6 +208,20 @@ export class RepoSyncState extends Model {
 		}));
 	}
 
+	static async findByOrgNameAndSubscriptionId(subscriptions: Subscription[], orgName: string): Promise<FlatArray<Awaited<RepoSyncState[]>[], 2>[]> {
+		const result = await Promise.all(subscriptions.map(async sub => {
+			return await RepoSyncState.findAll({
+				where: {
+					subscriptionId: sub.id,
+					repoOwner: {
+						[Op.iLike]: `%${orgName}%`
```

Comment:
I chatted with Darcy about this. Expected behaviour is that users will type either part of an org name, or the whole thing, so we need to do partial matches. I had some concerns around race conditions so asked him what they were doing to try and avoid that. He told me they use Lodash to debounce requests from the client and ensures that the last request made is the one that will be shown to the user.

Reply:
/nit
Can you put something in your list that `May need to put index on col repoOwner to speed up performance`? 
But I think we may not need that until really future.